plutus-benchmark/nofib/src/PlutusBenchmark/NoFib/Knights/Sort.hs

```diff
@@ -99,9 +101,9 @@ randomIntegers s1 s2 =
         if 1 <= s2 && s2 <= 2147483398 then
             rands s1 s2
         else
-            error "randomIntegers: Bad second seed."
+            error () -- "randomIntegers: Bad second seed."
```

Comment:
Is there a reason not to use `traceError "randomIntegers: Bad second seed."`?

Reply:
Good idea